### PR TITLE
feat(landingPage):Add LandingPage component

### PR DESCRIPTION
**Pull request by carlosbritto32**
## Summary
A component with all landing Page layout was added to the repo.
this solves the issue #40

### CODE REVIEW

File: apps/booking-app/pages/landingPage.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { LandingPage } from "@booking-ui/shared";
+
+export default function Home(){
+  return(<LandingPage/>)
+}
```
Comment (alexcamachogz) [inline apps/booking-app/pages/landingPage.js:5]: Add EOL

**Review by gh0stl1m — COMMENTED** [review]
@carlosbritto32 pretty good job so let's add the tests and with that, we can be able to merge the PR.

**Comment by carlosbritto32** [comment]
I just did a new [PR](https://github.com/Platzi-Master-C9/booking-ui/pull/151) because, i had trouble to renderize my component, in this PR i solved the renderize problem..

**Comment by Cristhian-Medina** [comment]
This issue was resolved in other PR

**Pull request closed without merging**